services/account.rst

```diff
@@ -0,0 +1,105 @@
+Account policy and procedures
+******
+
+This page presents PFS account policy and procedures on registration, update, 
+or contacts on trouble. 
+
+First of all, if you are a member of the institutes in the PFS collaboration, 
+you need an account for access to SuMIRe/PFS PBworks wiki page. 
+You can go to `Request access <http://sumire.pbworks.com/w/request-access>`_
+page and request an account. 
+
+After you have access to this SuMIRe/PFS PBworks wiki, your needs of other 
+accounts and services depend on your role in the PFS project as explained 
+in `Technical team member`_ and `Science team member`_. 
+Follow the instructions and request appropriate account(s). 
+
+Notes and tips:
+
+* If you already have some of the accounts and plan to request others, 
+  please read `Notes and tips for the registration process`_.
+* If you lost your account information, refer 
+  `Lost account or change password`_. 
+* The contents in the services for PFS software development are 
+  open to the public (apart from a part of them in the private repositories), 
+  so they are readable without any account. 
+
+Technical team member
+------
+
+If you are committing to works for the instrument development as a member 
+in the technical team of the PFS project, you need to get an account to use 
+`services of pfs.ipmu.jp (https) <https://pfs.ipmu.jp>`_ at least. 
+Optionally you need access to 
+software development tools (like `PFS JIRA <https://pfspipe.ipmu.jp/jira/>`_ 
+or `Subaru-PFS GitHub <https://github.com/Subaru-PFS>`_) if you work 
+on software development. Ask the local site manager at your institution 
+to send the following information to pfsprojectoffice at ipmu.jp.
+
+* Your contact email address
+* Preferred account name
+* Photo for photo directory at pfs.ipmu.jp (JPEG, less than 50kB; width 250-300 pixels, height 250-350 pixels)
+* If you will work on software development (in any area of PFS software development) 
+
+  * Request of access to PFS JIRA
+  * Your GitHub account
+
+Once your request is granted, the account information for pfs.ipmu.jp (https) 
+is emailed, so you must change your password from the initial one delivered 
+by the email. If you need to reset your password, contact pfs at pfs.ipmu.jp.
```

Comment:
first one is for creating account. password reset is just administrative operation.
or might be better to remove reset password part since we have 'Lost account or change password' section?

Reply:
So, creating account has management aspect.. I see.
How about linking to "Lost account change (reset, by the way?) password" section?

Reply:
ah, saying the same thing, but that would be a point. > to link